smslibrary/src/main/java/com/eis0/smslibrary/SMSMessage.java

```diff
@@ -1,34 +1,75 @@
 package com.eis0.smslibrary;
 
+import androidx.annotation.NonNull;
+
+/**
+ * Class implementing the Message interface, it represent a message of type SMS.
+ *
+ * @author Marco Cognolato
+ */
 public class SMSMessage implements Message {
+
+    // Public needed for test, but is also good to let the user know
+    public static final int MAX_MESSAGE_LENGTH = 160;
```

Comment:
Piccola nota: il gruppo 4 ha notato che in realtà è tipo sui 155 caratteri, non sappiamo come mai, meglio aggiornare forse?

Reply:
Non so che test abbia fatto il gruppo 4 per determinare il limite, però vorrei far notare che loro utilizzano come APP_ID un carattere non presente nella [tabella GSM-7](https://www.developershome.com/sms/gsmAlphabet.asp), che per l'appunto è il 0x02 ASCII. Per questo motivo c'è bisogno di più di un byte per codificare quel carattere e di conseguenza il massimo numero di caratteri diminuisce.
Noi utilizziamo come APP_ID il carriage return, che è presente nella tabella GSM-7, di conseguenza il limite dovrebbe essere (dico dovrebbe perché non l'ho testato) ancora i 160 caratteri classici.